bigchaindb/events.py

```diff
@@ -67,7 +67,7 @@ def get_subscriber_queue(self, event_types=None):
         """
 
         try:
-            self.started_queue.get_nowait()
+            self.started_queue.get(timeout=0.01)
```

Comment:
A bigger timeout (say, a second) won't hurt - a successful build will take the same amount of time anyway while  if there is an issue, waiting a second is not a big deal.

Reply:
I really don't know what would be an appropriate value of this timeout i.e. 10ms, 100ms, 1sec? But I do understand the having no timeout at all would lead us back to the situation wherein the test failed randomly. Also one must keep in mind that while the timeout will help the test case it will also impact the code performance.

Reply:
Since CI often fails for us here it makes sense to stay on the safe side and set a second. It only impacts the performance of the unsuccessful case. Do you expect failed retrievals to happen a lot? I suppose, we should not expect them to happen often so it's ok to wait for a second if it happens.

Reply:
> Do you expect failed retrievals to happen a lot?

The only place where `get_subscriber_queue` is called in the code base is [here](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/start.py#L53). So it seems that it might not effect much.